### PR TITLE
requests code review

### PR DESCRIPTION
**Pull request by ebukaume**

### CODE REVIEW

File: app/controllers/events_controller.rb [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    @events = Event.all
+  end
+
+  private
+
+  def event_params
+    params.required(:event).permit(:description, :date)
+  end
+
+  def ensure_unique_event
```
[the remark this answers is not in the document]
Comment (ebukaume) [reply app/controllers/events_controller.rb]: That's true, I forgot it's a validation task; thanks.

**Comment by ebukaume** [comment]
My partner was having internet issues at his home and he couldn't push code with his phone.

However, we stayed in contact most of the time and he made some good contributions too.

File: app/models/user.rb [hunk 2]
```diff
@@ -0,0 +1,29 @@
+class User < ApplicationRecord
+  before_save { name.capitalize! }
+  
+  validates :name, presence: true, length: {within: 3..50}
+  validates_uniqueness_of :name
```
Comment (mcrd25) [inline app/models/user.rb]: This is cool, but realistically I am thinking name would not be unique, you'd be forcing someone with the same name as someone else to change their name. Maybe username, or better yet, you should add an email attribute!

**Comment by DenisCuencaBrex** [comment]
Hi Maya, how are u? While we made the project i was without internet, so we talked a lot in slack and i contributed to the project there. 
We used email at first but we resorted to using only name for login since that's what the project stated we should use, as we didn't want to go against the project direcives and we are planing to refactor all the front end after React course while Made ours portfolios.

**Comment by mcrd25** [comment]
Okay, fair enough 😅. My guidelines conflict slightly with this. We'd need to update these guidelines soon. 🙂

**Comment by mcrd25** [comment]
What about the tests?

**Comment by ebukaume** [comment]
Hi @mcrd25,

I am personally going to take you up on your offer; specifically with regards to capybara DSL as well as Model test best practices.

>  You forgot about your unit and model tests as mentioned in project description under additional Microverse requirements using RSpec. If you need additional material on RSpec... feel free to contact me

Thank you.

**Comment by mcrd25** [comment]
Hi, @ebukaume and @DenisCuencaBrex 👋,

I mentioned unit and model tests, I meant unit and integration tests. Either way, these were additional requirements in pathwright, please do not forget for next project add the additional project requirements to get the 👍 .

Also, please remember this is particularly important for your Rails Final Project!

-- Maya

**Pull request merged**